packages/react-dom/src/__tests__/ReactDOMRoot-test.js

```diff
@@ -471,6 +471,27 @@ describe('ReactDOMRoot', () => {
     );
   });
 
+  it('return ReactDOMRoot if container is element', () => {
+    const root = ReactDOMClient.createRoot(document.createElement('div'));
+    expect(root._internalRoot.containerInfo.constructor.name).toEqual(
```

Comment:
`_internalRoot` is an internal property. The tests primarily focus on testing public APIs. What public API are you trying to test that isn't currently tested?

Reply:
Added few test to React DOM-
> - First one checks where the `ReactDOMClient createRoot` executes .
> - second one checks to find rendered component with the type in document.